commands/games/overwatch.js

```diff
@@ -1,363 +1,47 @@
-    }
-
-    if (quickplayPlayedEl !== null) {
-      played.quickplay = quickplayPlayedEl.trim().replace(/,/g, "");
-    }
-
-    if (quickplayTimePlayedEl !== null) {
-      time.quickplay = quickplayTimePlayedEl.trim().replace(/,/g, "");
-    }
-
-    if (compWonEl !== null) {
-      won.competitive = compWonEl.trim().replace(/,/g, "");
-    }
-
-    if (compLostEl !== null) {
-      lost.competitive = compLostEl.trim().replace(/,/g, "");
-    }
-
-    if (compDrawEl !== null) {
-      draw.competitive = compDrawEl.trim().replace(/,/g, "");
-    }
-
-    if (compPlayedEl !== null) {
-      played.competitive = compPlayedEl.trim().replace(/,/g, "");
-    }
-
-    if (compTimePlayedEl !== null) {
-      time.competitive = compTimePlayedEl.trim().replace(/,/g, "");
-    }
-
-    if (starEl !== null) {
-      star = $(".player-level .player-rank").attr("style").slice(21, 107);
-    }
-
-    const json = {
-      username: user,
-      level: parseInt(gameLevel) + prestigeLevel,
-      portrait: portrait,
-      games: {
-        quickplay: {
-          won: parseInt(won.quickplay),
-          played: parseInt(played.quickplay) || undefined
-        },
-        competitive: {
-          won: parseInt(won.competitive),
-          lost: parseInt(lost.competitive),
-          draw: parseInt(draw.competitive) || 0,
-          played: parseInt(played.competitive)
-        },
-      },
-      playtime: { quickplay: time.quickplay, competitive: time.competitive },
-      competitive: { rank: parseInt(compRank), rank_img: compRankImg },
-      levelFrame: levelFrame,
-      star: star
-    };
-
-    const profile = json,
-      jsonUsername = profile.username, // eslint-disable-line no-unused-vars
-      jsonLevel = profile.level,
-      jsonPortrait = profile.portrait, // eslint-disable-line no-unused-vars
-      jsonQpGamesWon = profile.games.quickplay.won,
-      jsonQpGametime = profile.playtime.quickplay,
-      jsonCompGamesWon = profile.games.competitive.won,
-      jsonCompGamesLost = profile.games.competitive.lost,
-      jsonCompGamesDraw = profile.games.competitive.draw,
-      jsonCompGamesPlayed = profile.games.competitive.played,
-      jsonCompGametime = profile.playtime.competitive,
-      jsonCompRank = profile.competitive.rank,
-      jsonCompRankImg = profile.competitive.rank_img, // eslint-disable-line no-unused-vars
-      jsonLevelFrame = profile.levelFrame, // eslint-disable-line no-unused-vars
-      jsonStar = profile.star; // eslint-disable-line no-unused-vars
-
+  async run(message, [platform, location, player], level) { // eslint-disable-line no-unused-vars
+    if (!player.length || !location.length || !platform.length) return message.response(undefined, `Ba....Baka! Invalid Usage, please do:\`${this.help.usage}\``);
+    if (!locationArray.includes(location)) return message.response(undefined, "Ba....Baka! Invalid Location. Valid Locations are us eu, kr, cn or global");
```

Comment:
While the PR is good some touch ups could be done, add ❌ to the response instead of undefined

Reply:
@MrJacz It auto adds it in message.response 🤔